config.yaml.defaults

```diff
@@ -26,6 +26,11 @@ server:
     # this should be set to False.
     ssl: False
 
+    # If you are using certificates, you can specify them here
+    ssl_use_certificate: False
```

Comment:
Please add a link to the nginx docs, showing what these arguments should be. Then good to go.

Reply:
Added!